apps/landing/components/Terms/Content.tsx

```diff
@@ -0,0 +1,394 @@
+        <Typography
+          tag="h2"
+          weight="bold"
+          className="text-natural-950 text-[1.5rem] sm:text-[2rem]"
+        >
+          {intl("terms/body/conversations-title")}
+        </Typography>
+        <Typography
+          tag="p"
+          weight="medium"
+          className="text-natural-950 text-[1rem] sm:text-[1.5rem]"
+        >
+          {intl("terms/body/conversations-content")}
+        </Typography>
+      </div>
+
+      <div className="flex flex-col gap-4 sm:gap-6">
+        <Typography
+          tag="h2"
+          weight="bold"
+          className="text-natural-950 text-[1.5rem] sm:text-[2rem]"
+        >
+          {intl("terms/body/rating-tutors-title")}
+        </Typography>
+        <Typography
+          tag="p"
+          weight="medium"
+          className="text-natural-950 text-[1rem] sm:text-[1.5rem]"
+        >
+          {intl("terms/body/rating-tutors-content")}
+        </Typography>
+      </div>
+
+      <div className="flex flex-col gap-4 sm:gap-6">
+        <Typography
+          tag="h2"
+          weight="bold"
+          className="text-natural-950 text-[1.5rem] sm:text-[2rem]"
+        >
+          {intl("terms/body/contact-us-title")}
+        </Typography>
+        <Typography
+          tag="p"
+          weight="medium"
+          className="text-natural-950 text-[1rem] sm:text-[1.5rem]"
+        >
+          {intl("terms/body/contact-us-content")}
+        </Typography>
+      </div>
+
+      <div className="flex flex-col gap-4">
+        <div className="flex flex-row gap-2">
+          <Typography
+            tag="h3"
+            weight="bold"
+            className="text-natural-950 text-[1rem] sm:text-[1.5rem]"
+          >
+            {intl("terms/body/contact-us/email/1")}
+          </Typography>
+          <Typography
+            tag="h3"
+            weight="medium"
+            className="text-natural-950 text-[1rem] sm:text-[1.5rem]"
+          >
+            {intl("terms/body/contact-us/email/2")}
+          </Typography>
+        </div>
+
+        <div className="flex flex-row gap-2">
+          <Typography
+            tag="h3"
+            weight="bold"
+            className="text-natural-950 text-[1rem] sm:text-[1.5rem]"
+          >
+            {intl("terms/body/contact-us/phone/1")}
+          </Typography>
+          <Typography
+            tag="h3"
+            weight="medium"
+            className="text-natural-950 text-[1rem] sm:text-[1.5rem]"
+          >
+            {intl("terms/body/contact-us/phone/2")}
+          </Typography>
+        </div>
+
+        <div className="flex flex-col sm:flex-row flex-wrap gap-4">
+          <Typography
+            tag="h3"
+            weight="bold"
+            className="text-natural-950 text-[1rem] sm:text-[1.5rem]"
+          >
+            {intl("terms/body/contact-us/social-media")}
+          </Typography>
+
+          <div className="flex gap-2">
+            <a
+              href="/instagram"
```

Comment:
These not our social media links. Check https://linktr.ee/litespace_eng for links.